### PR TITLE
LPD-2141 Adapt UserIndexerTest testSummaryHighlight depeding on the elasticsearch version

### PR DESCRIPTION
**Pull request by gustavolimav**
In elasticsearch 8.10.2 the behaviour add a change to highlighter type unified that caused a behaviour change in user summary highlight

https://liferay.atlassian.net/browse/LPD-2141

https://liferay.atlassian.net/browse/LPD-2141?focusedCommentId=2587344

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:report:2542058392

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote

**Comment by gustavolimav** [comment]
ci:test:search-opensearch2

**Comment by gustavolimav** [comment]
ci:report:2543887495

**Comment by gustavolimav** [comment]
ci:report:2543882436

**Comment by gustavolimav** [comment]
ci:report:2543882417

**Comment by gustavolimav** [comment]
ci:report:2543880757

**Comment by gustavolimav** [comment]
need to consider opensearch

**Comment by gustavolimav** [comment]
solr is fine

**Comment by gustavolimav** [comment]
failed with es8 suit

**Comment by gustavolimav** [comment]
search suit passed

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote-minimum

**Comment by gustavolimav** [comment]
ci:report:2544164306

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote-minimum

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote-minimum

**Comment by gustavolimav** [comment]
ci:report:2544709494

**Comment by gustavolimav** [comment]
Tests with es8 looks good, I will put this ticket in review before run the other tests

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/indexer/test/UserIndexerTest.java [hunk 1]
```diff
@@ -856,6 +855,45 @@ protected void toggleCreateSite(Organization organization, boolean site)
...
+	}
+
+	private String _getExpectedFullNameHighlight(
+		String firstName, String middleName, String lastName) {
+
+		String expectedTitleHighlight = StringBundler.concat(
+			HighlightUtil.HIGHLIGHT_TAG_OPEN, firstName,
+			HighlightUtil.HIGHLIGHT_TAG_CLOSE, StringPool.SPACE, middleName,
+			StringPool.SPACE, HighlightUtil.HIGHLIGHT_TAG_OPEN, lastName,
+			HighlightUtil.HIGHLIGHT_TAG_CLOSE);
```
Comment (AlmirFe) [inline modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/indexer/test/UserIndexerTest.java:881]: You can inline it. Do not need to create a variable for this. See https://github.com/AlmirFe/liferay-portal/commit/9648821efa67586db3e2756a7e81fd70bb2ed825

**Review by AlmirFe — CHANGES_REQUESTED** [review]
Hi @gustavolimav, I only requested a small change. The rest of the code looks fine to me.

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/indexer/test/UserIndexerTest.java (hunk 1 above)
Comment (gustavolimav) [reply modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/indexer/test/UserIndexerTest.java:881]: Thx Almir! I will Cherry pick it

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote-minimum

**Comment by gustavolimav** [comment]
ci:test:search-es8-remote

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:search-solr

**Review by AlmirFe — APPROVED** [review]
LGTM! :rocket:

**Comment by gustavolimav** [comment]
es8

**Comment by gustavolimav** [comment]
ci:report:2545135867

**Comment by gustavolimav** [comment]
es7

**Comment by gustavolimav** [comment]
ci:report:2545137643

**Comment by gustavolimav** [comment]
ci:report:2545141864

**Comment by gustavolimav** [comment]
solr

**Comment by gustavolimav** [comment]
opensearch

**Comment by gustavolimav** [comment]
es8 - OK
es7- OK
es8-minumum- OK
solr- OK
opensearch - OK

**Comment by gustavolimav** [comment]
ci:forward

**Pull request closed without merging**